draft-ietf-dnsop-generalized-notify.md

```diff
@@ -540,14 +566,16 @@ delegations seems disproportionately costly, while infrequent scanning
 causes slower convergence (delay until the delegation is updated).
 
 While use of the NOTIFY mechanism for coordinating the key exchange in
```

Comment:
Perhaps: "...use of the NOTIFY mechanism for coordinating the key exchange in multi-signer setups (where the parties may be identified via lookup of the DSYNC RRset)..."

Or perhaps not. Never mind.